pages/index.js

```diff
@@ -75,7 +75,7 @@ const BigFontStyle = styled.div`
 `;
 
 const BigFontStyle2 = styled.div`
-  margin-top: 1.5em;
+  margin-top: 1em;
```

Comment:
The name "BigFontStyle2" does not fit the CSS property "margin", right?

Reply:
I don´t understand your comment here. The name "BigFontStyle2" refers to the second big font on my homepage.